sixtyfps_runtime/corelib/model.rs

```diff
@@ -498,11 +507,31 @@ impl<T> core::cmp::PartialEq for ModelRc<T> {
 }
 
 impl<T> ModelRc<T> {
-    pub fn new(model: Rc<dyn Model<Data = T>>) -> Self {
+    pub fn new(model: impl Model<Data = T> + 'static) -> Self {
+        Self(Some(Rc::new(model)))
+    }
+}
+
+impl<T, M: Model<Data = T> + 'static> From<Rc<M>> for ModelRc<T> {
```

Comment:
`From<&Rc<M>>` would be another variant we could offer. Not sure if worth it though.

Reply:
We can always add that later if we need.